src/components/Events/EventCard.js

```diff
@@ -69,57 +73,59 @@ const AlignRightCol = styled(Col)`
 `
 
 const EventCard = ({ event }) => {
+  // TODO delete the default value for type and attendees when they can be retrieved
   const {
-    type,
     date,
-    eventName,
-    eventLocation,
+    address,
     startTime,
     endTime,
-    attendees = 0,
-    link
+    type = 'TODO - type',
+    eventTitle,
+    attendees = `TODO - ${0}`,
```

Comment:
For the moment we can remove this and just conditionally render the attendees, no need to show `0`